<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MazeProblem</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mazeWidget" native="true">
      <property name="styleSheet">
       <string notr="true">
QWidget
{
    color: white;
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="setFrame">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame{
    color: white;
    background-color: rgb(250, 250, 250);
    border: 5px solid rgb(250, 250, 250);
    border-radius:0px;
    border-color: rgb(250, 250, 250);
    font:10pt &quot;微软雅黑&quot;;
	color:rgb(0, 0, 0)
}</string>
      </property>
      <widget class="QRadioButton" name="tenSize">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>10X10</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="twentySize">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>20X20</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="thirtySize">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>40X40</string>
       </property>
      </widget>
      <widget class="QPushButton" name="createMazeButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(0, 170, 255);
border:none;
font: bold 9pt &quot;微软雅黑&quot;;
color:rgb(255,255,255);
border-radius: 5px;
}
QPushButton::hover{
background-color:rgb(0, 150, 225);
}
QPushButton::pressed{
background-color:rgb(0, 150, 225);
}
</string>
       </property>
       <property name="text">
        <string>显示迷宫</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="threeRank">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>普通</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="sixRank">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>简单</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="zeroRank">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>复杂</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>68</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>规模</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>复杂度</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AStarButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(0, 170, 255);
border:none;
font: bold 9pt &quot;微软雅黑&quot;;
color:rgb(255,255,255);
border-radius: 5px;
}
QPushButton::hover{
background-color:rgb(0, 150, 225);
}
QPushButton::pressed{
background-color:rgb(0, 150, 225);
}
</string>
       </property>
       <property name="text">
        <string>A*算法求路径</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="showProcessButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>87</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>显示过程</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ACOButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(0, 170, 255);
border:none;
font: bold 9pt &quot;微软雅黑&quot;;
color:rgb(255,255,255);
border-radius: 5px;
}
QPushButton::hover{
background-color:rgb(0, 150, 225);
}
QPushButton::pressed{
background-color:rgb(0, 150, 225);
}
</string>
       </property>
       <property name="text">
        <string>ACO算法求路径</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
